fivejars/openy_gated_content
# Fix for Category/Video pages

Resolves https://github.com/ymcatwincities/openy_gated_content/issues/56. Fix by @bharath-kondeti

To test:
- Ensure that Videos and Blog posts have different categories
- Click between the Video listing and Blog listing using the main navigation
- Observe the page content changes
